python/coral_usb/detector_base.py

```diff
@@ -395,3 +397,77 @@ def config_callback(self, config, level):
         config = super(EdgeTPUPanoramaDetectorBase, self).config_callback(
             config, level)
         return config
+
+
+class EdgeTPUTileDetectorBase(EdgeTPUPanoramaDetectorBase):
```

Comment:
```suggestion
class EdgeTPUTileDetectorBase(EdgeTPUDetectorBase):
```

Do we need to inherit `EdgeTPUPanoramaDetectorBase`? `EdgeTPUDetectorBase` is correct?